ext_localconf.php

```diff
@@ -228,13 +228,13 @@
         \T3G\AgencyPack\Blog\Hooks\DataHandlerHook::class;
 
     /** @noinspection UnsupportedStringOffsetOperationsInspection */
-    $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['ext/install']['update']['T3G\AgencyPack\Blog\Install\Updates\DatabaseMonthYearUpdate']
+    $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['ext/install']['update'][\T3G\AgencyPack\Blog\Updates\DatabaseMonthYearUpdate::class]
```

Comment:
This is a breaking change which will cause this wizard to be re-executed. You should fix the identifier returned by the wizards instead.

Reply:
I talked to @helsner already about this in private chat, I forgot about this and will close it now.

Reply:
What was the result of this talk?